go-controller/pkg/ovndbmanager/ovndbmanager.go

```diff
@@ -167,6 +173,10 @@ func ensureClusterRaftMembership(db string, kclient kube.Interface) {
 	var appCtl func(args ...string) (string, string, error)
 	clusterStatusRetryCnt := &nbClusterStatusRetryCnt
 
+	// IPv4 example: tcp:172.18.0.2:6641
+	// IPv6 example: tcp:[fc00:f853:ccd:e793::3]:6642
+	dbServerRegexp := `(ssl|tcp):(\[?([a-z0-9.:]+)\]?:\d+)`
```

Comment:
just regex compile this once, rather than every iteration in the for loop, you can use regexp.MustCompile